packages/tools/pie-wrapper-react/react-gen-wrapper-config.js

```diff
@@ -0,0 +1,16 @@
+import genReactWrapper from './scripts/react-gen-wrapper.js';
+import path from 'path';
+import fs from 'fs-extra';
+
+const componentPath = path.resolve(process.cwd(), './custom-elements.json')
+
+// fetches custom-elements.json file
+const loadJSON = (path) => JSON.parse(fs.readFileSync(path));
+
+export default {
+    plugins: [
```

Comment:
The same comment goes for `react-remove-wrapper-config`: why is it being exported as an object? Are these files being used anywhere else?

Reply:
ended up removing this file 👍 see comment below